src/main/resources/assets/gilly7ce-carpet-addons/lang/en_us.json

```diff
@@ -15,6 +15,7 @@
   "carpet.rule.netheritePickaxeInstantMineEndStone.desc": "A netherite pickaxe with efficiency V combined with the haste II status effect will instant mine end stone type blocks.",
   "carpet.rule.netheritePickaxeInstantMineNetherBricks.desc": "A netherite pickaxe with efficiency V combined with the haste II status effect will instant mine nether brick type blocks.",
   "carpet.rule.phantomsObeyHostileMobCap.desc": "Phantoms will no longer spawn if the hostile mobcap is full. This is per player.",
+  "carpet.rule.replaceableFlowers.desc": "placing blocks on flowers will replace them like grass.",
```

Comment:
Capitalize the first word in the description.

Reply:
It was written like this in issue reee